.bazelrc

```diff
@@ -129,4 +129,11 @@ test --test_output=errors
 # warn if a timeout is much longer than actual runtime
 test --test_verbose_timeout_warnings
 
+# Use --config=lldb to run a test under lldb
+# to connect run `gdb-remote 7654` in an lldb session
+test:lldb --run_under='@llvm_18_toolchain_llvm//:bin/lldb-server g :7654 --'
```

Comment:
>We can't directly connect using lldb due
to bazel controlling stdin/stdout/stderr

interesting. can lldb terminal session connect to a remote so it is as-if directly attached?

Reply:
Yeah it's working well for me.

Reply:
There are still some quirks with the sanitizers, but I can step through code